addons/l10n_tr_nilvera_einvoice/models/account_move.py

```diff
@@ -49,6 +49,80 @@ class AccountMove(models.Model):
         copy=False,
         default='not_sent',
     )
+    l10n_tr_gib_invoice_scenario = fields.Selection(
+        selection=[
+            ('TEMELFATURA', "Basic"),
+            ('KAMU', "Public Sector"),
+        ],
+        default='TEMELFATURA',
+        string="Invoice Scenario",
+        help="The scenario of the invoice to be sent to GİB.",
+    )
+    l10n_tr_gib_invoice_type = fields.Selection(
+        compute='_compute_l10n_tr_gib_invoice_type',
+        store=True,
+        readonly=False,
+        default='SATIS',
+        string="GIB Invoice Type",
+        selection=[
+            ('SATIS', "Sales"),
+            ('TEVKIFAT', "Withholding"),
+            ('IHRACKAYITLI', "Registered for Export"),
+            ('ISTISNA', "Tax Exempt"),
+        ],
+        help="The type of invoice to be sent to GİB.",
+    )
+    l10n_tr_is_export_invoice = fields.Boolean(string="GİB Product Export Invoice")
+    l10n_tr_shipping_type = fields.Selection(
+        selection=[
+            ('1', "Sea Transportation"),
+            ('2', "Railway Transportation"),
+            ('3', "Road Transportation"),
+            ('4', "Air Transportation"),
+            ('5', "Post"),
+            ('6', "Combined Transportation"),
+            ('7', "Fixed Transportation"),
+            ('8', "Domestic Water Transportation"),
+            ('9', "Invalid Transportation Method"),
+        ],
+        string="Shipping Method",
+        help="The type of shipping.",
+    )
+    l10n_tr_exemption_code_id = fields.Many2one(
+        'l10n_tr_nilvera_einvoice.account.tax.code',
```

Comment:
Will not block for that, but good practice to always add the attribute you are changing so comodel_name=....